### DISCUSSION TITLE
Table of contents when importing MDX files

### DISCUSSION THREAD

**Discussion by MichaelStather** [facebook/docusaurus#7697] [Q&A]
Hello,

I have a question about the table of contents at the right side of the page.
What I see is that in documents consisting of a single .md file the TOC automatically appears.
But I also have documents which include other files, and they then include other files and so on.
Here no TOC appears.

What can I do?
Thnks for your help!

**Comment by slorber** [comment]
@MichaelStather it is unfortunately a painful limitation we currently have, tracked here: https://github.com/facebook/docusaurus/issues/3915
